inlong-sort/sort-core/src/test/java/org/apache/inlong/sort/parser/AllMigrateMongoDBTest.java

```diff
@@ -40,6 +40,7 @@
 import org.apache.inlong.sort.protocol.node.load.KafkaLoadNode;
 import org.apache.inlong.sort.protocol.transformation.FieldRelation;
 import org.apache.inlong.sort.protocol.transformation.relation.NodeRelation;
+import org.junit.Assert;
```

Comment:
Why import this class here?

Reply:
This PR  https://github.com/apache/inlong/pull/7998 caused OOM when running the workflow. Adding this is to test whether there is any impact.